docs/legal/dpa/subprocessors.rst

```diff
@@ -0,0 +1,62 @@
+Sub-Processor List
+==================
+
+:Effective: April 16, 2021
+:Last updated: April 16, 2021
+
+Read the Docs for Business uses services from the following sub-processors to
+provide documentation hosting services. This document supplements :doc:`our Data
+Processing Agreement <index>` and may be separately updated on a periodic basis.
+A sub-processor is a third party data processor who has or potentially will have
+access to or will process personal data.
+
+.. seealso::
+    Previous versions of this document, as well as the change history to this
+    document, are available `on GitHub`_
+
+.. _on GitHub: https://github.com/readthedocs/readthedocs.org/commits/master/docs/legal/dpa/subprocessors.rst
+
+Infrastructure
+--------------
+
+Amazon Web Services, Inc.
+    Cloud infrastructure provider.
+
+Services
+--------
+
+Elasticsearch B.V.
+    Hosted ElasticSearch services for documentation search. Search indexes do
+    not include user data.
+
+Sendgrid, Inc.
+    Provides email delivery to dashboard and admin users for site notifications
+    and other generated messages. The body of notification emails can include
+    user information, including email address.
+
+Google Analytics
+    Website analtyics for dashboard and documentation sites.
+
```

Comment:
Stripe?

Reply:
Yeah, just realized looking at the policy doc that stripe is missing from the list.